src/AddTodoForm.jsx

```diff
@@ -1,22 +1,30 @@
-const AddTodoForm = (props) => {
+import { useState } from 'react';
+
+const AddTodoForm = ({onAddTodo}) => {
+    let [todoTitle, setTodoTitle] = useState("");
+
+    const handleTitleChange = (event) => {
+        let todoInput = document.querySelector("input");
+        let newTodoTitle = todoInput.value;
+        //let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    };
+
     const handleAddTodo = (event) => {
+        // let form = event.target;
```

Comment:
Make sure you're removing unused code comments. Additionally, I am unsure why you are using `let` instead of `const` while declaring variables. While this is not technically incorrect, most of the values that we work with are non-mutating. It's best practice to declare them with `const`so that the code is easier to read and it is more difficult to introduce bugs. Please have these updated with your 1.6 lesson PR submission.

[More info on var, let, const](https://www.freecodecamp.org/news/differences-between-var-let-const-javascript/)

Reply:
I have replaced `let` with `const` in all the variable declarations which I have in my code, thus far. Then, I tested the code in the browser to make sure that it still works; and it does. I have also moved any important information that I had in my comments into the README.md file, so that I would not lose it when I removed the comments from my code. Then, per your request, I removed all the comments from my code.